roles/info/tasks/openshift.yml

```diff
@@ -0,0 +1,83 @@
+- name: Create config map with fetched VMware data
+  kubernetes.core.k8s:
+    definition:
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: vmware-viewer
+        namespace: default
+      data:
+        data.json: "{{ __apliances.results | map(attribute='appliance') | to_json }}"
+    apply: yes
+
+- name: Create deployment with vmware data
+  kubernetes.core.k8s:
+    definition:
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        labels:
+          app: vmware-viewer
+        name: vmware-viewer
+        namespace: default
+      spec:
+        replicas: 2
+        selector:
+          matchLabels:
+            app: vmware-viewer
+        template:
+          metadata:
+            labels:
+              app: vmware-viewer
+          spec:
+            volumes:
+              - name: config-volume
+                configMap:
+                  name: vmware-viewer
+            containers:
+              - image: quay.io/machacekondra/vmware_viewer
```

Comment:
actually it might be cool to optionally create an nginx image or template a Dockerfile that serves this information. that would allow people running it on something other than openshift to deploy the results as well (im thinking locally, like docker desktop or minikube but it could be pushed to some other kubernetes system or like ECS)

Reply:
Yeah, that makes a lot of sense. Most of the parameters here should definitely be paramaterized. I will for now move out the code out of here and add it as a separate PR.

The image is image with react application which parses the JSON and shows it in the UI. Anyone could write it's custom app to show the json data, so definitelly this should be possible to change.